source/05-pages/homepage.stories.jsx

```diff
@@ -59,7 +59,7 @@ const homepageContent = args => twigTemplate({
   homepage_grid_title: 'You Don’t Vote For Kings',
 });
 
-const Homepage = args => <PageWrapper>{parse(homepageContent(args))}</PageWrapper>;
+const Homepage = args => <PageWrapper front="true">{parse(homepageContent(args))}</PageWrapper>;
```

Comment:
I think `isHomepage` is a better variable name. Also, we don’t need to add `="true"` here.

Recommended:
```
const Homepage = args => <PageWrapper isHomepage>{parse(homepageContent(args))}</PageWrapper>;
```

Reply:
This should be `front={true}` assuming the intent is to set front as a boolean, not set it to the string "true."

Reply:
Or just go with Dan's suggestion. I'm good with that too.